### PR TITLE
Refactor layout

### PR DESCRIPTION
**Pull request by ShrawanBK**

### CODE REVIEW

**Comment by AdityaKhatri** [comment]
@ShrawanBK , in general, your commit messages should reflect what it does in terms of the project and not only in terms of this PR.
Going through your last 2 commits, they are minor changes that don't need a separate commit, so you need to merge them to the same one in this case.
For future reference, the commit message `Refactor Layout` would have been better if it was `Refactor report page layout` as it points to something more specific.

File: client/src/views/Report/GroupedBarChart.js [hunk 1]
```diff
@@ -17,6 +17,13 @@ export default function GroupedBarChart(props) {
         data,
     } = props;
 
+
+    if (data.values.length <= 0) {
```
Comment (AdityaKhatri) [inline client/src/views/Report/GroupedBarChart.js:21]: Are you sure data.values is defined?

File: client/src/views/Report/GroupedBarChart.js [hunk 2]
```diff
@@ -25,13 +32,17 @@ export default function GroupedBarChart(props) {
 
     const barLabels = getBarLabels(data);
 
+    const numberOfData = data.values.length;
```
Comment (AdityaKhatri) [inline client/src/views/Report/GroupedBarChart.js:35]: Are you sure data.values is defined?

File: client/src/views/Report/GroupedBarChart.js (hunk 1 above)
Comment (ShrawanBK) [reply client/src/views/Report/GroupedBarChart.js:21]: The `getSoiTrendData` has
```return ({
            values,
            columns: [
                'Total Closed',
                'Closed On',
            ],
            colors: {
                'Total Closed': '#41cf76',
                'Closed On': '#ef8c00',
            },
        })
```
and data is the returned value of above calculation. So, I think data.values will be defined.

Do, I need to check if data.values exists as well ?

**Review by samshara — APPROVED** [review]

**Pull request merged**